packages/storybook/mock/GrowthBookProvider.tsx

```diff
@@ -0,0 +1,8 @@
+import { fn } from '@storybook/test';
+import * as actual from '@dailydotdev/shared/src/components/GrowthBookProvider';
+
+export * from '@dailydotdev/shared/src/components/GrowthBookProvider';
+
+export const useFeature = fn(actual.useFeature)
+  .mockName('useFeature')
+  .mockReturnValue('control');
```

Comment:
This will mock all useFeature to `control`? it might impact some other values in that case

Reply:
I would like to add an initial value, just for common cases, but it is editable in each component story! Should I remove it?

Reply:
Hmm if we can somehow set it for each story that be great?
Since you're one is the only one exposing it for now, we can probably go ahead like this.